lib/rbac/roles.rb

```diff
@@ -16,6 +16,17 @@ def with_each_role
       end
     end
 
+    def self.assigned_role?(role_name)
```

Comment:
Should we have a method to get all role names in an array? In the code we have `admin?` and `approver?` that will required to make RBAC service call twice.